Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1068,7 +1068,7 @@ class BrowserViewController: UIViewController {
 
     func openBlankNewTabAndFocus(isPrivate isPrivate: Bool = false) {
         popToBVC()
-        openURLInNewTab(nil, isPrivate: isPrivate)
+        openURLInNewTab(nil, isPrivate: isPrivate, isPrivileged: true)
```

Comment:
Edit: Nevermind; I think `nil` falls back to the user's homepage choice, which could be local.